test/com/google/javascript/jscomp/ProcessCommonJSModulesTest.java

```diff
@@ -251,38 +251,4 @@ public void testEs6ObjectShorthand() {
         "goog.provide('module$test');"
             + "module$test = { prop: 'value', foo() { console.log('bar'); }}" );
   }
-
-  public void testSortInputs() throws Exception {
-    SourceFile a = SourceFile.fromCode("a.js", "require('b');require('c')");
```

Comment:
I believe we do - we great test coverage for sorting: https://github.com/google/closure-compiler/blob/master/test/com/google/javascript/jscomp/JSModuleGraphTest.java

The CommonJS modules created their own module graph and this test was for that. I removed that code as it didn't work well with other modules and as far as I can tell was completely unnecessary.